drivers/timer/hpet.c

```diff
@@ -224,30 +239,26 @@ static inline void hpet_timer_comparator_set(uint32_t val)
 #define HPET_CMP_MIN_DELAY		(1000)
 #endif
 
-#define MAX_TICKS			0x7FFFFFFFUL
-
 static __pinned_bss struct k_spinlock lock;
-static __pinned_bss unsigned int last_count;
+static __pinned_bss uint64_t last_count;
 
 #ifdef CONFIG_TIMER_READS_ITS_FREQUENCY_AT_RUNTIME
 static __pinned_bss unsigned int cyc_per_tick;
-static __pinned_bss unsigned int max_ticks;
 #else
 #define cyc_per_tick			\
 	(CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC / CONFIG_SYS_CLOCK_TICKS_PER_SEC)
-
-#define max_ticks			\
-	((MAX_TICKS - cyc_per_tick) / cyc_per_tick)
 #endif /* CONFIG_TIMER_READS_ITS_FREQUENCY_AT_RUNTIME */
 
+#define max_ticks ((int32_t)0x7fffffff)
```

Comment:
Since you are removing the max_ticks variable above, just make the macro in capital letters at it should be.

Reply:
done.